etc/schema.yaml

```diff
@@ -62,7 +62,8 @@ run_elements:
         mean_insert_size: { type: 'float' },
         std_dev_insert_size: { type: 'float' },
         median_insert_size: { type: 'integer' },
-        median_abs_dev_insert_size: { type: 'integer' }
+        median_abs_dev_insert_size: { type: 'integer' },
+        other_insert_type: { type: 'dict' }
```

Comment:
Is there a more descriptive name than `other_insert_type`? At the moment, we're referring to inserts that are not forward-reverse facing - something like non_facing_inserts, maybe?

Reply:
I guess that is debatable and I don't have very strong preferences.